test/ppx_deriving_qcheck/deriver/qcheck2/test_sig.ml

```diff
@@ -0,0 +1,42 @@
+open QCheck2
+open Helpers
+
+(* Here we check that deriving qcheck2 works in signature, and cover the cases
+   where the type is named "t" and when it is not. *)
+
+module T : sig
+  type t = int [@@deriving_inline qcheck2]
+    include sig
+      [@@@ocaml.warning "-32"]
+      val gen : t QCheck2.Gen.t 
+    end
+    [@@ocaml.doc "@inline"]
+  [@@@deriving.end]
+
+  type string' = string [@@deriving_inline qcheck2]
+    include sig
+      [@@@ocaml.warning "-32"]
+      val gen_string' : string' QCheck2.Gen.t 
+    end
+    [@@ocaml.doc "@inline"]
+  [@@@deriving.end]
+end = struct
+  type t = int [@@deriving qcheck2]
+
+  type string' = string [@@deriving qcheck2]
+end
+
+let test_int () =
+  test_compare ~msg:"Gen.int <=> deriving int" ~eq:Alcotest.int Gen.int T.gen
+
+let test_string () =
+  test_compare ~msg:"Gen.string <=> deriving string" ~eq:Alcotest.string Gen.string T.gen_string'
+
+(** {2. Execute tests} *)
+
+let () = Alcotest.run "Test_Primitives"
+           [("Primitives",
+             Alcotest.[
+                 test_case "test_int" `Quick test_int;
+                 test_case "test_unit" `Quick test_string;
```

Comment:
All 6 test titles here seem copy-pasted from `test_primitives.ml` (which is a natural starting point).
To separate their outcome in the log from the other ones, it would make sense to include the keyword `"signature"` in them.

Reply:
Ah, I am not familiar with alcotest, and had missed the naming pattern in use. I've made an attempt to improve based on your comment. Thank you!